Split PHPCR 3.0 upgrade steps in more steps (#7894)
sulu/sulu: UPGRADE-3.x.md

## 3.0.0

The upgrade from Sulu 2.6 to Sulu 3.0 is a major upgrade and will require some migration steps.

### Pre Update step to 3.0 PHPCR Migration

Before upgrading to 3.0 make sure you have installed the latest Sulu 2.x versions of Sulu and the Sulu bundles.
Make sure that you have all PHPCR migration run via:

```shell
php bin/adminconsole phpcr:migrations:migrate
```

### Pre Update step to 3.0 Sulu Article Bundle

Due to the fact that in Sulu 3.0 the SuluArticleBundle is merged into the core Sulu package, before updating the

> `sulu/sulu` dependency, you have to update the sulu/article-bundle dependency to the latest version.

Reviewer: ```suggestion
Due to the fact that in Sulu 3.0 the SuluArticleBundle is merged into the core Sulu package, if you have the
sulu/article-bundle installed, you need to update the sulu/article-bundle dependency to the latest version before
updating the sulu/sulu dependency. If the sulu/article-bundle is not installed, this step is not necessary.
```

    target:
        dbal:
            connection: default
```

> Make sure you have done all database migrations below for 3.0 after that you can run the following command to update the content structure:

Reviewer: ```suggestion
Please ensure that you have executed all the database migrations for version 3.0. Once finished, you 
can run the following command to update the content structure:
```

        </property>
```

### Upgrade the Controller references

> The controller in the page/article templates has to be adjusted use the new controller

Reviewer: ```suggestion
The controller in the page/article templates have to be adjusted use the new controller
```
